cmd/dockerd/daemon.go

```diff
@@ -284,28 +284,32 @@ func (cli *daemonCLI) start(ctx context.Context) (err error) {
 
 	c, err := createAndStartCluster(cli, d)
 	if err != nil {
-		log.G(ctx).Fatalf("Error starting cluster component: %v", err)
+		return errors.Wrap(err, "error starting cluster component")
```

Comment:
Wondering if we want to both log and return this one (not sure if the error shows up in logs?)

edit: answering myself; it looks like we do this already for som other errors further up. Looking it such errors were logged, I noticed that for _windows_ we explicitly send it to the logs (but only when running as service), but for _unix_ we don't;

on Windows; https://github.com/moby/moby/blob/2f6953fcd8a411816ea84f4d24fd9ea101c19ba7/cmd/dockerd/docker_windows.go#L28-L33

on Linux; https://github.com/moby/moby/blob/2f6953fcd8a411816ea84f4d24fd9ea101c19ba7/cmd/dockerd/docker_unix.go#L12-L14

Reply:
huh, I don't remember changing this line of code.
Maybe I saw it was `return fmt.Errorf(...)` with a `%v` and not properly wrapped...